upcoming/composable-pipelines-pattern.md

```diff
@@ -0,0 +1,392 @@
++++
+Author = ["Samuel Lampa"]
+date = "2015-01-30T00:00:00-08:00"
+series = [""]
+title = "Pattern for pluggable pipeline components in Go"
++++
+
+# Patterns for composable concurrent pipelines in Go
```

Comment:
Oops, should be "Patterns for composable concurrent pipelines in Go"
Den 7 feb 2015 20:39 skrev "Nathan Youngman" <notifications@github.com>:

> In upcoming/composable-pipelines-pattern.md
> <https://github.com/gopheracademy/gopheracademy-web/pull/114#discussion_r24292092>
> :
>
> > @@ -0,0 +1,392 @@
> > ++++
> > +Author = ["Samuel Lampa"]
> > +date = "2015-01-30T00:00:00-08:00"
> > +series = [""]
> > +title = "Pattern for pluggable pipeline components in Go"
> > ++++
> > +
> > +# Patterns for composable concurrent pipelines in Go
>
> You're title here and above don't match.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gopheracademy/gopheracademy-web/pull/114/files#r24292092>
> .
>